src/js/common/api/signApi.js

```diff
@@ -0,0 +1,83 @@
+const codeitPath = "https://bootcamp-api.codeit.kr/api";
+import { accessTokenValid } from "../util/storage.js";
+
+/**
+ * 로그인 POST
+ */
+async function signInPost(inputValue) {
+  try {
+    const response = await fetch(`${codeitPath}/sign-in`, {
+      method: "POST",
+
+      body: JSON.stringify(inputValue),
+
+      headers: {
+        "Content-Type": "application/json",
+      },
+    });
+
+    if (response.status === 400) {
+      throw new Error(`HTTP ERROR CODE : ${response.status}`);
+    }
+    const { data } = await response.json();
+    localStorage.setItem("accessToken", data.accessToken);
+    window.location.href = "/folder.html";
+    return response.ok;
+  } catch (e) {
+    console.log(e);
+  }
+}
+
+/**
+ * 이메일 중복 확인 POST
+ */
+async function emailValidkPost(inputValue) {
```

Comment:
```suggestion
async function emailValidPost(inputValue) {
```

typo가 조금씩 보이는데, spell check plugin을 사용해보시면 좋겠네요!

Reply:
수정완료했습니다.

Code Spell Checker 플러그인을 사용중인데 Valid 구문자체를 오타로 인식하는줄 알았습니다. 
좀더 자세히 살펴봐야겠네요 ㅎㅎ